eng/build.ps1

```diff
@@ -212,7 +212,7 @@ function BuildSolution() {
   $projects = Join-Path $RepoRoot $solution
   $toolsetBuildProj = InitializeToolset
 
-  $testTargetFrameworks = if ($testCoreClr) { "netcoreapp3.1" } else { "" }
+  $testTargetFrameworks = if ($testCoreClr) { 'net5.0%3Bnetcoreapp3.1' } else { "" }
```

Comment:
Why not just use "net5.0"?

Reply:
Similar to the above. The IDE is on the cusp of adding `netcoreapp3.1` tests into the repository. Given I'd already figured out the magic spell to run both TF correctly I kept it in here. Both of these changes were kept after some offline discussions with IDE (needed to understand their direction a bit better).